test/spec/modules/aolBidAdapter_spec.js

```diff
@@ -544,12 +544,11 @@ describe('AolAdapter', () => {
 
       expect(formattedPixels).to.equal(
         '<script>var w=window,prebid;' +
-        'for(var i=0;i<10;i++){w = w.parent;prebid=w.$$PREBID_GLOBAL$$;' +
+        'try{for(var i=0;i<10;i++){w = w.parent;prebid=w.$$PREBID_GLOBAL$$;' +
```

Comment:
1.0 compliant adapters are not permitted to reference $$PREBID_GLOBAL$$ and not permitted to drop their own pixels.  Looks like this code has been here awhile but is not it is not compliant with the 1.0 adapter rules

Reply:
@mike-chowla 
Hi. Thank you for reviewing.

It has been there since we updated our adapter for 1.0.  In migration PR we provided more details about this feature (https://github.com/prebid/Prebid.js/pull/1693) and it was accepted.

Actually, we don't drop own pixels. It is exactly the same pixels that we drop via `getUserSyncs` but attached to the ad content.

Reply:
I actually rejected the initial pull-request partially because of the custom pixels and it doesn't look like it was resolved before being merged.  I still feel the same way I did then but I'm fine with being overruled; I'll assign @mkendall07 to review and see with how he feels (if this needs to be fixed now or not).